litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -425,6 +425,10 @@ func (r *queryResolver) ListDashboard(ctx context.Context, projectID string) ([]
 	return analyticsHandler.QueryListDashboard(projectID)
 }
 
+func (r *queryResolver) PortalDashboardData(ctx context.Context, projectID string, hubName string) ([]*model.PortalDashboardData, error) {
```

Comment:
Can you please add RBAC check here
`err := authorization.ValidateRole(ctx, projectID, []model.MemberRole{model.MemberRoleOwner, model.MemberRoleEditor, model.MemberRoleViewer}, usermanagement.AcceptedInvitation)
	if err != nil {
		return nil, err
	}`
Like this. You can modify the roles as per the requirement

Reply:
Sure @SarthakJain26 , will update the same. Thanks!